narwhals/dataframe.py

```diff
@@ -561,7 +581,7 @@ def schema(self) -> dict[str, DType]:
             ...         "ham": ["a", "b", "c"],
             ...     }
             ... )
-            >>> df = nw.DataFrame(df_pl)
+            >>> df = nw.from_native(df_pl)
             >>> df.schema  # doctest: +SKIP
```

Comment:
it's good that you're using `from_native` here, but the docstring should be updated to follow the template described in the issue

Reply:
Fixed

Reply:
Fixed